chat/be/package.json

```diff
@@ -8,6 +8,7 @@
     "dotenv": "^10.0.0",
```

Comment:
It looks like you ran npm install at some point.  Please remove chat/be/package-lock.json and make sure to use yarn in the future.